stable/aws-for-fluent-bit/values.yaml

```diff
@@ -109,6 +109,8 @@ elasticsearch:
 #     Format template
 #     Template {time} used={Mem.used} free={Mem.free} total={Mem.total}
 
+rbac:
+  create: true
```

Comment:
And just remove this new configuration option...

Reply:
Yes